tools/releaseBuild/azureDevOps/templates/release-PublishPackageMsftCom.yml

```diff
@@ -9,19 +14,40 @@ steps:
   displayName: Set Package version
 
 - pwsh: |
-   git clone https://$(AzureDevOpsPat)@mscodehub.visualstudio.com/PowerShellCore/_git/Internal-PowerShellTeam-Tools '$(Pipeline.Workspace)/tools'
+    Write-Verbose -verbose "git clone --verbose https://$(AzureDevOpsPat)@mscodehub.visualstudio.com/PowerShellCore/_git/Internal-PowerShellTeam-Tools '$(Pipeline.Workspace)/tools'"
+    git clone --verbose https://$(AzureDevOpsPat)@mscodehub.visualstudio.com/PowerShellCore/_git/Internal-PowerShellTeam-Tools '$(Pipeline.Workspace)/tools'
   displayName: Clone Internal-PowerShellTeam-Tools from MSCodeHub
 
+- pwsh: |
+    sudo apt update
+    sudo apt install -y python3 python3-pip python-is-python3
+  displayName: Install Python
+
+- task: PipAuthenticate@1
+  inputs:
+    artifactFeeds: 'pmc'
+    pythonDownloadServiceConnections: pmcDownload
+
+- pwsh: |
+    pip install pmc-cli
+
+    $newPath = (resolve-path '~/.local/bin').providerpath
+    $vstsCommandString = "vso[task.setvariable variable=PATH]${env:PATH}:$newPath"
+    Write-Host "sending " + $vstsCommandString
+    Write-Host "##$vstsCommandString"
+  displayName: Install pmc cli
+  
 - pwsh: |
     $metadata = Get-Content -Path "$(Build.SourcesDirectory)/tools/metadata.json" -Raw | ConvertFrom-Json
-    Write-Verbose -Verbose "$(ReleaseTag) $(RepoClientCliClientID) $(RepoClientCliSecret) $(AzureVersion) $($metadata.LTSRelease.Latest)"
     $params = @{
         ReleaseTag = "$(ReleaseTag)"
         AadClientId = "$(RepoClientCliClientID)"
         AadClientSecret = "$(RepoClientCliSecret)"
         BlobFolderName = "$(AzureVersion)"
         LTS = $metadata.LTSRelease.Latest
         ForProduction = $true
+        SkipPublish = $${{ parameters.skipPublish }}
     }
+    $params | Out-String -width 9999 | write-Verbose -Verbose
     & '$(Pipeline.Workspace)/tools/packages.microsoft.com/releaseLinuxPackages.ps1' -MappingFilePath '$(System.DefaultWorkingDirectory)/tools/packages.microsoft.com/mapping.json'  @params
```

Comment:
perhaps we should change the path in the tools repo so when it's merged we don't break the servicing branches